app/helpers/auth/registrations_helper.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+module Auth::RegistrationsHelper
+  def registrations_new_confirm_password_input_html
```

Comment:
If this is to be extracted to a new helper, I think it should have a less confusing name explicitly referencing this is a honeypot field.

Reply:
Added honeypot to naming.